### ISSUE TITLE
improve timeline to include all personal activities

### ISSUE THREAD

**Issue by ionphractal** [Human-Connection/WebApp#41]
When I make comments/reaction/... I would like them to be included in my personal activity stream as a history, where I can go back and see what I've done so far and where I participated and so on.

There should also be a setting to change visibility (self only, friends, public) of my activities that happened in other than my own posts.

**Comment by sebako86** [comment]
that's a good point. We need a general setting for all future posts and a way to allow visibility setting on a per post/per comment level.

**Comment by ionphractal** [comment]
Thanks! Please let me know if you need anything else from me in this regard.

**Comment by appinteractive** [comment]
is implemented in the usersettings branch and will go live with it.

**Comment by sebako86** [comment]
comment functions are still very basic and probably a bit of buggy at parts.

**Comment by appinteractive** [comment]
@ionphractal yeah it is ordering by post date not comment date. Good point. We will look into it. The timeline stuff is indeed very basic and will be worked on a lot in the next time.
